packages/sdk/src/react/ui/modals/_internal/components/currencyOptionsSelect/index.tsx

```diff
@@ -49,24 +55,23 @@ const CurrencyOptionsSelect = observer(function CurrencyOptionsSelect({
 		(currency) =>
 			({
 				label: currency.symbol,
-				value: currency.contractAddress,
+				value: currency.symbol,
```

Comment:
I dont think we should change this.
For some chains we have the same symbol on multiple currencies, the user would not know what to pick, but at least the app do not break